docs/src/main/sphinx/release/release-401.md

```diff
@@ -0,0 +1,42 @@
+# Release 401 (19 Oct 2022)
+
+## General
+
+* Add support for using path-style access for all requests to S3 when using
+  fault-tolerant execution with exchange spooling. This can be enabled with the
+  `exchange.s3.path-style-access` configuration property. ({issue}`14655`)
+* Add output buffer utilization distribution to `EXPLAIN ANALYZE VERBOSE`. ({issue}`14596`)
+* Add operator blocked time distribution to `EXPLAIN ANALYZE VERBOSE`. ({issue}`14640`)
+
+## Delta Lake connector
+
+* Add support for writing to tables using [version 4 of the writer](https://docs.delta.io/latest/versioning.html#features-by-protocol-version).
+  This does not yet include support for [change data feeds](https://docs.delta.io/2.0.0/delta-change-data-feed.html)
+  or generated columns. ({issue}`14573 `)
+* Fix bug where executing a `COMMENT` or `ALTER TABLE ... ADD COLUMN` statement
```

Comment:
Maybe 

* Avoid changing reader and writer version table properties as unrelated side effect of adding comments.

Reply:
fair .. sorry .. I misread.. .. leave the statements then..